packages/firebase_analytics/firebase_analytics/lib/src/firebase_analytics.dart

```diff
@@ -6,8 +6,7 @@ part of firebase_analytics;
 
 /// Firebase Analytics API.
 class FirebaseAnalytics extends FirebasePluginPlatform {
-  FirebaseAnalytics._({required this.app})
-      : super(app.name, 'plugins.flutter.io/firebase_analytics');
+  FirebaseAnalytics._({required this.app}) : super(app.name, 'plugins.flutter.io/firebase_analytics');
```

Comment:
I guess this (and other similar changes) depends on specific IDE settings and could be reversed.

Reply:
https://github.com/firebase/flutterfire/pull/11030/commits/2ff81c0ca3bf3e85b82ad8feec89da2be3d08b4a